Assets/LeapMotion/Core/Resources/HandData.cginc

```diff
@@ -0,0 +1,311 @@
+﻿// HandData.cginc
+
+// ==============
+// Fingertip Data
+// ==============
+
+uniform float4 _Leap_LH_Fingertips[5];
+uniform float4 _Leap_RH_Fingertips[5];
+
+uniform float4 _Leap_LH_PalmTriangles[15];
+uniform float4 _Leap_RH_PalmTriangles[15];
+
+uniform float4 _Leap_LH_FingerSegments[28];
+uniform float4 _Leap_RH_FingerSegments[28];
+
+// =========
+// Functions
+// =========
+
+#define IDENTITY_MATRIX4x4 { 1.0, 0.0, 0.0, 0.0, \
+                             0.0, 1.0, 0.0, 0.0, \
+                             0.0, 0.0, 1.0, 0.0, \
+                             0.0, 0.0, 0.0, 1.0 }
+
+float4x4 Leap_HandData_Preprocess_Matrix = IDENTITY_MATRIX4x4;
+
+void Leap_ClearPreprocessMatrix() {
```

Comment:
Should this method just get removed?  It's a no-op

Reply:
ye